resources/geocoding/en/93.txt

```diff
@@ -0,0 +1,52 @@
+# Copyright (C) 2012 The Libphonenumber Authors 
+
+# Licensed under the Apache License, Version 2.0 (the "License"); 
+# you may not use this file except in compliance with the License. 
+# You may obtain a copy of the License at 
+ 
+# http://www.apache.org/licenses/LICENSE-2.0 
+ 
+ 
+# Unless required by applicable law or agreed to in writing, software 
+# distributed under the License is distributed on an "AS IS" BASIS, 
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+# See the License for the specific language governing permissions and 
+# limitations under the License. 
+# Generated from: 
+# https://www.itu.int/dms_pub/itu-t/oth/02/02/T02020000010002PDFE.pdf [2008-08-28] 
```

Comment:
Ah you're right! I missed that there are several communications in that document. Please keep this date :)

Some mobile numbers are actually geocoded (e.g. in the United States), but that's not the case here.